src/pybind/mgr/progress/module.py

```diff
@@ -451,7 +464,7 @@ def __init__(self, *args, **kwargs):
         super(Module, self).__init__(*args, **kwargs)
 
         self._events = {}  # type: Dict[str, Union[RemoteEvent, PgRecoveryEvent, GlobalRecoveryEvent]]
-        self._completed_events = [] # type: List[GhostEvent]
+        self._completed_events = []  # type: List[GhostEvent]
```

Comment:
this is not part of this change.

Reply:
just making the space consistent with the line above